src/Microsoft.NET.Sdk.Functions.MSBuild/Targets/netstandard1.0/Microsoft.NET.Sdk.Functions.Publish.targets

```diff
@@ -30,6 +30,7 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     -->
   <Target Name="_OverrideFunctionsPack"
           AfterTargets="Pack"
+          Condition="'$(DisablePublishDuringPack)' != 'true'"
```

Comment:
Ideally this should be conditioned on IsPackable. So, if IsPackable is set to true then pack is enabled and publish is disabled but IsPackable needs more changes to work correctly. This is a work-around until then.

Reply:
LGTM